plannerbenchmark/postProcessing/metrics.py

```diff
@@ -40,42 +62,67 @@ def computeMetric(self, data):
 
 
 class TimeToReachGoalMetric(Metric):
+    """Metric to compute the time it took to reach the goal.
+
+    Requires the threshold, `des_distance`.
+    """
+
     def computeMetric(self, data):
-        m = self._params['m']
+        m = self._params["m"]
```

Comment:
'm' can be given a more representative name. It appears in multiple other places such as exp.yml and experiment.py